examples/nidmm/continuous-acquire-graph-multiple-points.py

```diff
@@ -12,6 +12,9 @@
 # For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
 # Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
 #
+# Refer to the NI DMM Support Manual page to determine the valid channel and resource names for your NI-DMM module.
+# Link : https://zone.ni.com/reference/en-XX/help/370384V-01/TOC151.htm
```

Comment:
Remove this link. This is not the right link. All of ni's documentation would be moving to a new experience on ni.com (not the link you have pasted). However right now it is unlikely that the MI documentation would be moving by Q2. Hence we should go ahead with our plan of converting chm to mark down & submitting them in the wiki.
You can either retain that placeholder wiki link you had in your previous commit, or you could remove the link & add it once the documentation story is done (it is anyway captured as part of this task - https://dev.azure.com/ni/DevCentral/_workitems/edit/1420181)

Reply:
removed the link